Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationMessageWindowTableViewAdapter+DataSource.swift

```diff
@@ -107,6 +107,29 @@ extension ConversationMessageWindowTableViewAdapter: ZMConversationMessageWindow
                 self.tableView.lockContentOffset = false
             }
         }
+        
+        if changeInfo.insertedIndexes.count > 0 {
```

Comment:
Wouldn't it be possible to do:

```suggestion
        if changeInfo.insertedIndexes.count > 0 || isLoadingInitialContent {
```

and then remove `selectLastMessage` in `viewDidLoad`, then everything would be contained inside this method.

Reply:
I see ok.